desktop_files/package.json.orig

```diff
@@ -67,7 +67,7 @@
     "react-native-image-crop-picker": "0.18.1",
     "react-native-image-resizer": "1.0.0",
     "react-native-invertible-scroll-view": "1.1.0",
-    "react-native-keychain": "git+https://github.com/status-im/react-native-keychain.git#v.3.0.0-status",
+    "react-native-keychain": "git+https://github.com/status-im/react-native-keychain.git#v.3.0.0-status-1",
```

Comment:
I suggest standardizing on the following format `v.3.0.0-1-status` (keeping `-status` as the suffix). I've been using `v` prefix if the base repo also uses that format for their own releases.

It looks like you didn't create a release for `v.3.0.0-status-1`, is that correct?

Reply:
@PombeirP, I created it

Reply:
Oh, I see, you created a release named `v3.0.0-status-1` (without the dot after `v`)

Reply:
@PombeirP ,let me create a new tag with suggested format.